cattle/plugins/docker/compute.py

```diff
@@ -314,10 +314,10 @@ def get_container(self, client, instance, by_agent=False):
         if instance is None:
             return None
 
-        name = '/{0}'.format(instance.uuid)
         container_list = client.containers(all=True, trunc=False)
         container = self.find_first(container_list,
-                                    lambda x: self._name_filter(name, x))
```

Comment:
Also, we are getting all containers and the iterating until we find a match (looking through each container's labels and then its). Is it worth adding a filter on the uuid label?